MFVNeutralino/python/EventFilter_cfi.py

```diff
@@ -1,21 +1,23 @@
 import FWCore.ParameterSet.Config as cms
 from JMTucker.Tools.PATTupleSelection_cfi import jtupleParams
 
+#FIXME : update lepton minpT but need to study min_njets and min_ht
 mfvEventFilter = cms.EDFilter('MFVEventFilter',
```

Comment:
@abbywarden resolved?  I don't think we want any min_ht (as you are doing). Njets I could see an argument that each lq pair should have a jet, but also would want to see that it matters before applying any selection on two jets...